elastic-caller.php

```diff
@@ -9,7 +9,7 @@
  * @return array of appointments
  */
 function elastic_request( $api_path, $search_param ) {
-	$url    = 'http://hosting.rotaract.de:9200' . $api_path;
+	$url    = ROTARACT_ELASTIC_HOST . $api_path;
```

Comment:
Shouldn't be there a check whether `ROTARACT_ELASTIC_HOST` is set?
Something like

```php
if ( ! defined( 'ROTARACT_ELASTIC_HOST' ) ) {
  return DUMMY_OBJECT_THAT_PREVENTS_PHP_ERROR_AND_CRASH_DOWNS;
}
```